routes/camera.js

```diff
@@ -0,0 +1,117 @@
+import express from "express";
+const router = new express.Router();
+import http from "http";
+
+router.get("/save/:id(\\d\\d?)/", (req, res) => {
+    let id = req.params.id;
+    if (id.length === 1) {
+        id = `0${id}`;
+    }
+    http.get({
+        host: "192.168.10.101",
```

Comment:
Would it be useful to put the host address in a global variable so that when it changes, you don't have to change it in multiple locations?